### PR TITLE
Moves 'web' target to the gulpfile.

### PR DESCRIPTION
**Pull request by yurydelendik**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1157,6 +1169,66 @@ gulp.task('importl10n', function(done) {
...
+         .pipe(gulp.dest(GH_PAGES_DIR)),
+    gulp.src([FIREFOX_BUILD_DIR + '*.xpi',
+              FIREFOX_BUILD_DIR + '*.rdf'])
+        .pipe(gulp.dest(GH_PAGES_DIR + EXTENSION_SRC_DIR + 'firefox/')),
+    gulp.src([CHROME_BUILD_DIR + '*.crx',
+              FIREFOX_BUILD_DIR + '*.rdf'])
+        .pipe(gulp.dest(GH_PAGES_DIR + EXTENSION_SRC_DIR + 'chromium/')),
+    gulp.src('test/features/**/*', {base: 'test/'})
+        .pipe(gulp.dest(GH_PAGES_DIR)),
+    gulp.src(JSDOC_DIR + '**/*', {base: JSDOC_DIR})
```
Comment (Snuffleupagus) [inline gulpfile.js]: I was puzzled as to why this folder, https://github.com/mozilla/pdf.js/tree/gh-pages/api/draft, wasn't copied over.

It seems that should add `var JSDOC_BUILD_DIR = BUILD_DIR + 'jsdoc/';` with the other constants at the top of the file, and then use that here instead.
Bonus points if this new constant is also used in https://github.com/mozilla/pdf.js/blob/master/gulpfile.js#L932 as well :-)

File: gulpfile.js [hunk 2]
```diff
@@ -1157,6 +1169,66 @@ gulp.task('importl10n', function(done) {
...
+
+  // 'vinyl' because web/viewer.html needs its BOM.
+  return merge([
+    vinyl.src(GENERIC_DIR + '**/*', {base: GENERIC_DIR, stripBOM: false})
+         .pipe(gulp.dest(GH_PAGES_DIR)),
+    gulp.src([FIREFOX_BUILD_DIR + '*.xpi',
+              FIREFOX_BUILD_DIR + '*.rdf'])
+        .pipe(gulp.dest(GH_PAGES_DIR + EXTENSION_SRC_DIR + 'firefox/')),
+    gulp.src([CHROME_BUILD_DIR + '*.crx',
+              FIREFOX_BUILD_DIR + '*.rdf'])
```
Comment (Snuffleupagus) [inline gulpfile.js]: I know that you're just moving the old code over, but after looking at this line:
Why are we copying files, relevant to the `FIREFOX` addon, into a folder that ought to contain the `CHROMIUM` extension!?

File: gulpfile.js [hunk 3]
```diff
@@ -1157,6 +1169,66 @@ gulp.task('importl10n', function(done) {
...
+
+gulp.task('gh-pages-git', ['gh-pages-prepare', 'wintersmith'], function () {
+  var VERSION = getVersionJSON().version;
+  var reason = process.env['PDFJS_UPDATE_REASON'];
+
+  spawnSync('git', ['init'], {cwd: GH_PAGES_DIR});
+  spawnSync('git', ['remote', 'add', 'origin', REPO], {cwd: GH_PAGES_DIR});
+  spawnSync('git', ['add', '-A'], {cwd: GH_PAGES_DIR});
+  spawnSync('git', [
+    'commit', '-am', 'gh-pages site created via make.js script',
```
Comment (Snuffleupagus) [inline gulpfile.js]: Nit: Replace `make.js` with `gulpfile.js` here.

File: gulpfile.js [hunk 4]
```diff
@@ -48,11 +50,15 @@ var COMPONENTS_DIR = BUILD_DIR + 'components/';
 var SINGLE_FILE_DIR = BUILD_DIR + 'singlefile/';
 var MINIFIED_DIR = BUILD_DIR + 'minified/';
 var FIREFOX_BUILD_DIR = BUILD_DIR + 'firefox/';
+var CHROME_BUILD_DIR = BUILD_DIR + '/chromium/';
```
Comment (Snuffleupagus) [inline gulpfile.js]: Nit: For consistency with the other lines, the first slash can be removed.

Comment (timvandermeij) [inline gulpfile.js]: Remove the extra slash before `chromium` for consistency with the other variables above it.

**Review by Snuffleupagus — APPROVED** [review]
Looks good, thank you!

r=me, with the comments addressed.

File: gulpfile.js (hunk 2 above)
Comment (timvandermeij) [inline gulpfile.js]: This does not look right. Why would the Chome extension folder need to contain the Firefox `.rdf` file?

File: gulpfile.js [hunk 5]
```diff
@@ -1157,6 +1169,66 @@ gulp.task('importl10n', function(done) {
...
+        .pipe(gulp.dest(GH_PAGES_DIR + EXTENSION_SRC_DIR + 'chromium/')),
+    gulp.src('test/features/**/*', {base: 'test/'})
+        .pipe(gulp.dest(GH_PAGES_DIR)),
+    gulp.src(JSDOC_DIR + '**/*', {base: JSDOC_DIR})
+        .pipe(gulp.dest(GH_PAGES_DIR + 'api/draft/')),
+  ]);
+});
+
+gulp.task('wintersmith', ['gh-pages-prepare'], function (done) {
+  mkdirp.sync(GH_PAGES_DIR);
```
Comment (timvandermeij) [inline gulpfile.js]: Shouldn't this already exist because `gh-pages-prepare` is a prerequisite here?

**Review by timvandermeij — CHANGES_REQUESTED** [review]

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Nice work, thanks!

**Pull request merged**